src/style.css

```
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-style: bold;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.firstForm {
+  margin-top: 20px;
+  position: relative;
+}
+
+.secondForm {
+  margin: 20px 0;
+  position: relative;
+}
+
+.search {
+  font-family: inherit;
+  color: rgba(61, 78, 97, 1);
+  font-size: 16px;
+  line-height: 22px;
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  border: 1px solid rgba(225, 231, 237, 1);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+}
+
+.icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+}
+
+.search--small {
+  font-size: 14px;
+  line-height: 19px;
+  height: 42px;
+  background: rgba(255, 255, 255);
```

Use rgba if you have alpha channel
```suggestion
  background: rgb(255, 255, 255);
```